src/CMakeLists.txt

```diff
@@ -127,8 +138,26 @@ add_library(caideInliner STATIC
 
 target_include_directories(caideInliner SYSTEM PRIVATE ${CLANG_INCLUDE_DIRS} ${LLVM_INCLUDE_DIRS})
 target_compile_definitions(caideInliner PRIVATE ${CLANG_DEFINITIONS} ${LLVM_DEFINITIONS})
-target_link_libraries(caideInliner PRIVATE
-    clangAST clangBasic clangFrontend clangLex clangRewrite clangSema clangTooling)
+
+if(CAIDE_INLINER_LINK_CLANG_DYLIB)
```

Comment:
Can you please use `CAIDE` (not `CAIDE_INLINER`) prefix for consistency with the existing option?

Reply:
Thanks for the tip. I will change the prefix of the options.